desktop/src/cli.rs

```diff
@@ -192,8 +192,8 @@ pub struct Opt {
     pub frame_rate: Option<f64>,
 
     /// The handling mode of links opening a new website.
```

Comment:
I don't think it's a great style to have an implicit default mode (confirm) that can't be explicitly selected.

Reply:
What do you mean? It can be selected by the user here or in preferences

Reply:
That's right!